test/resources/layout.css

```diff
@@ -152,6 +152,10 @@ body {
   width: 100%;
 }
 
+.input-playground--compact {
```

Comment:
Is this one used anywhere?

Reply:
it was used in backstop test template, as we removed test templates we're not using this now, I will remove.